src/main/java/Main.java

```diff
@@ -0,0 +1,53 @@
+import figure.*;
+
+public class Main {
+    public static void main(String[] args) {
+        final int FIGURES_COUNT = 4 + (int) (Math.random() * 10);
+        final String[] COLORS = {"red", "orange", "green", "black", "blue", "white", "yellow", "aqua", "purple", "navy", "lime", "violet"};
```

Comment:
this line is definitely longer than 100 chars, don't forget about codestyle

Reply:
sorry, forgot to use "mvn clean package". I`ve changed it